frontend/src/App.js

```diff
@@ -1,50 +1,98 @@
-import React from 'react';
+import React, { Component } from 'react';
```

Comment:
There's a need to import component?

Reply:
it just makes the coder nice so that you can declare `class App extends Component` instead of `class App extends React.Component`